.github/workflows/linter.yml

```diff
@@ -18,9 +18,7 @@ jobs:
           ${{ runner.os }}-gem-
     - name: Setup ruby
       uses: ruby/setup-ruby@v1
-    - run: |
-        gem install bundler
-        bundle config path vendor/bundle
-        bundle install --jobs 4 --retry 3
+    - name: Install dependencies
+      run: bundle install
```

Comment:
This step was failing as you can [see here](https://github.com/HiMamaInc/bambora-client/actions/runs/7735401383/job/21090993234?pr=51). It was failing to install the correct version of `bundler` for the given Ruby version.


The https://github.com/ruby/setup-ruby GitHub Action comes with [Bundler installed by default](https://github.com/ruby/setup-ruby#bundler), so we don't really need to re-install `bundler`. This ensures a correct version of `bundler` is installed, and now the tests and linting pass.